docs/deploying.md

```diff
@@ -69,119 +69,119 @@ Each service's `main` package is located in the `cmd` directory.
 | exposure cleanup | cmd/cleanup-exposure | Deletes old exposure keys |
 | export cleanup | cmd/cleanup-export | Deletes old exported files published by the exposure key export service |
 
-### Deploying Using Terraform configs
+### Deploying using Terraform
 
-The included terraform configs should enable you to bring up a copy of
-the exposure notifications server on GCP.  It makes heavy use of the GCP
-terraform provider, developed at
-https://github.com/terraform-providers/terraform-provider-google.
+You can use Terraform to deploy the reference Exposure Notification on Google
+Cloud. These instructions make use of the
+[Google Cloud Terraform provider](https://github.com/terraform-providers/terraform-provider-google).
 
-1. Download and install Terraform 0.12.  [Installation guide](https://www.terraform.io/downloads.html),
-although `go get github.com/hashicorp/terraform` may be all you need.
+1. Download and install Terraform 0.12.
 
-1.  Create a GCP project.
-    [Instructions](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
-    Enable a billing account for this project, and remember its project ID (the
-    unique, unchangeable string that you will be asked for during creation).
-    
-    ```text
-    $ export PROJECT_ID="<value-from-above>"
-     ```
+   To install Terraform using Go, type:
```

Comment:
You should _not_ do this. Terraform has pre-built binaries. Don't instruct users to compile from source.

Reply:
Removed as requested.